docs/data.md

```diff
@@ -0,0 +1,58 @@
+# Viewing Data
+
+## Directory Navigation
+### Moving up and down the tree
+Directories can be navigated through by clicking on an option in the navigation bar on the right side of the screen. To move back up the tree to a previous location simply select the location (or its parent) from the list of previous locations in the breadcrumbs at the top.
+
+![Navigate Folders](img/nav_folders.png)
+
+### Searching
+The search feature is currently only available within the `eSimMon-dashboard` directory within the `eSimMon` collection. This is the top level directory for the dashboard and is the location that you start at when first navigating to the page.
+
+As you begin searching the drop-down menu will update with options that match your search.
+
+![Search](img/search.png)
+
+A single item can be selected from the menu or you can press enter or click the magnifying glass icon to return matching results. If there are multiple items that match the results will be prepended with their full path in order to make it easier to differentiate between the options. Clearing the search bar will return you to the previous location.
+
+## Item Selection
+### Displaying data
+By default only 10 items are displayed at a time but this can be changed at the bottom of the navigation drawer.
```

Comment:
Do you think the term "drawer" will be familiar to the user? "menu" might be better?

Reply:
I think "menu" sounds good, I'll use that instead.